common/api/imodel-transformer.api.md

```diff
@@ -49,6 +73,12 @@ export class ChangedInstanceIds {
     element: ChangedInstanceOps;
     // (undocumented)
     font: ChangedInstanceOps;
+    // @beta
+    getCustomRelationshipDataFromId(id: Id64String): ChangedInstanceCustomRelationshipData | undefined;
```

Comment:
@aruniverse , @DanRod1999, can you advise me: 
`getCustomRelationshipDataFromId()` and `hasCustomRelationshipChanges()` are not meant to be called by third party code, those methods are supposed to be "internal". 
How to properly handle/mark such methods in 'imodel-transformer' package?I  feel we might want to remove them in future or implement in different way. Is it enough to mark them as 'beta'? Those will never be something that user should use.
I found one case where such internal method was marked as private and then called like this: 
https://github.com/iTwin/imodel-transformer/blob/4e4a7df0030ea7fba666c4b700dc6f4160074340/packages/transformer/src/IModelTransformer.ts#L3169

Reply:
ah i misread, dan is right